WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsConnectView.swift

```diff
@@ -0,0 +1,31 @@
+import UIKit
+
+final class CardReaderSettingsConnectView: UIView {
+    @IBOutlet var contentView: UIView!
+    @IBOutlet weak var connectButton: UIButton!
+
+    var onPressedConnect: (() -> ())?
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        sharedInit()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        sharedInit()
+    }
+
+    private func sharedInit() {
+        Bundle.main.loadNibNamed("CardReaderSettingsConnectView", owner: self, options: nil)
+        addSubview(contentView)
+        contentView.frame = self.bounds
+        contentView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
```

Comment:
Would it be possible to declare this using auto layout constraints instead of frames and autoresizing masks?